docs/source/index.rst

```diff
@@ -1,6 +1,8 @@
 
-llvmlite -- a light-weight Python binding to LLVM
-=================================================
+llvmlite
+========
+
+**A lightweight LLVM python binding for writing JIT compilers**
```

Comment:
It is not limited to writing JIT compilers.  I would just keep it as "A lightweight LLVM python binding".